tools/crmadmin.c

```diff
@@ -126,13 +116,9 @@ command_cb(const gchar *option_name, const gchar *optarg, gpointer data, GError
         command = cmd_list_nodes;
     }
 
-    if (!strcmp(option_name, "--election") || !strcmp(option_name, "-E")) {
-        command = cmd_elect_dc;
-    }
-
-    if (!strcmp(option_name, "--kill") || !strcmp(option_name, "-K")) {
-        command = cmd_shutdown;
-        crm_trace("Option %c => %s", 'K', optarg);
+    if (!strcmp(option_name, "--timeout") || !strcmp(option_name, "-t")) {
+        options.timeout = crm_parse_interval_spec(optarg);
```

Comment:
This will set errno (to EINVAL) if the user gave an unrecognizable value, so we should catch that and error out (glib was doing that for us when it was just an int)